src/components/feedback/test/Dialog-test.js

```diff
@@ -278,7 +284,7 @@ describe('Dialog', () => {
         // The onClose callback is not immediately invoked
         assert.notCalled(onClose);
         // Once the transition has ended, the callback should have been called
-        await delay(60); // Transition finishes after 50ms
+        await delay(70); // Transition finishes after 50ms
```

Comment:
This is a bit random, but the test was passing locally and failing in CI. I guess there's a chance there's a timing issue/race condition here, as we are talking about 10ms.